runners/dmn-tck-runner-drools/src/test/java/org/omg/dmn/tck/runner/drools/DroolsTCKTest.java

```diff
@@ -277,7 +331,17 @@ public void afterTest(Description description, TestSuiteContext context, TestCas
    @Override
    public void afterTestCase(TestSuiteContext context, TestCases testCases)
    {
-      // nothing to do
+      props.setProperty("last.update", ZonedDateTime.now().format(DateTimeFormatter.ISO_LOCAL_DATE));
+      try {
+         props.store(new FileOutputStream(propsFile), null);
+      } catch (IOException e) {
+         logger.warn("Exception while updating last update date", e);
+      }
+   }
+
+   @Override
+   public String getResultFileName() {
+      return "../../TestResults/Drools/" + props.getProperty("product.version") + "/tck_results.csv";
```

Comment:
This is breaking the API contract.

I suggest this line is fine, but to please include something like a `File.mkdirs()` in the DMN TCK abstract/base class?